Modules/Sources/Storage/GRDB/GRDBManager.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import GRDB
+
+public final class GRDBManager {
+
+    let databaseQueue: DatabaseQueue
```

Comment:
I chose DatabaseQueue instead of DatabasePool [based on the advice here](https://github.com/groue/GRDB.swift/?tab=readme-ov-file#database-connections) – a pool might end up being better for our needs, but let's see how we get on.